modules/bullet/godot_result_callbacks.cpp

```diff
@@ -51,11 +52,25 @@ bool GodotClosestRayResultCallback::needsCollision(btBroadphaseProxy *proxy0) co
 	if (needs) {
 		btCollisionObject *btObj = static_cast<btCollisionObject *>(proxy0->m_clientObject);
 		CollisionObjectBullet *gObj = static_cast<CollisionObjectBullet *>(btObj->getUserPointer());
+
+		AreaBullet *areaObj = dynamic_cast<AreaBullet *>(gObj);
```

Comment:
Use something else, as Android, HTML5 and other platforms are not compiled with RTTI to save size.

Reply:
You should use `Object::cast_to`.